private/buf/cmd/buf/command/convert/convert.go

```diff
@@ -52,32 +52,33 @@ Use an input proto to interpret a proto/json message and convert it to a differe
 
 The simplest form is:
 
-$ buf convert <input> --type=<type> --from=<payload> --to=<output>
+	$ buf convert <input> --type=<type> --from=<payload> --to=<output>
```

Comment:
Should be updated as well to make consistent

Reply:
So with the buf cli reference docs it would greatly simplify it if we were able to use indented code blocks. 
The other alternative is to create another `Example` field on appcmd which would allow formatting later on.

Reply:
I don't think we should invest further in solutions either here or in the buf cli reference (ie such as an Example field) - let's get this across the line, I would prefer things are not indented consistently, but either is fine so we can close this out.